### DISCUSSION TITLE
Projects: Select multiple values for sorting

### DISCUSSION THREAD

**Discussion by Craftplacer** [community/community#6262] [Projects and Issues] [answered]
While sorting is great, it would be better if you can select multiple values for sorting, and give each one a priority.

For example:
1. Status
2. Priority
3. Area
4. ...

This would keep things tidy

**Comment by Katolus** [comment]
Great suggestion!

**Comment by e-e** [comment]
Would also like to see this

**Comment by rileybroughten** [comment, marked as answer]
Closing this duplicate in favor of the top-voted discussion post https://github.com/orgs/community/discussions/8895 for this request.

Keep an eye on that post for updates! ✨
